README.md

```diff
@@ -1,46 +1,68 @@
 ## Purpose
 
-This process helps to mass produce Moderne abstract syntax trees (ASTs) for a large body of proprietary code without modifying the build processes of that code.
+One of the core steps for setting up Moderne is publishing Moderne abstract syntax trees (ASTs) to an artifact repository such as Artifactory or Nexus. 
 
-It is designed for use on a Jenkins installation with the [Job DSL](https://plugins.jenkins.io/job-dsl) plugin installed.
+While it is possible to [manually add a plugin to each repository](https://app.gitbook.com/o/-MEp_3EtccewzekKY8mZ/s/-MhFwm0iG8BFZKPYoFkH/how-to/integrating-private-code) to accomplish this, there are a few problems with that approach:
+* It does not scale well in organizations with many repositories.
+* It requires modifying the build processes of code, which may not be desired.
+* The task which generates the ASTs takes time and memory. This, in turn, could negatively affect the building and shipping of code.
 
-Each repository that is subject to nightly ingest is listed in `repos.csv`, along with calculated information about its build tooling and language level requirements.
+A much better and safer way to handle the creation of ASTs is to use Jenkins to mass produce the artifacts and for Moderne to ingest them nightly or at set intervals. Please note, though, that this could mean your ASTs are out of date if it's been a long time since the last ingest. 
 
-## Modifications to use in your environment
+## Requirements
```

Comment:
This is labeled as "requirements", but the first sentence talks about the approach. Maybe that sentence should go into the above section?

Reply:
Actually on second thought, this whole "Requirements" heading can go away, and the "Each repository that is subject ..." can be superseded by the setup instructions.